src/acl/Ip.cc

```diff
@@ -252,12 +252,80 @@ acl_ip_data::FactoryParse(const char *t)
         return q;
     }
 
-    /* Special ACL RHS "ipv6" matches IPv6-Unicast Internet */
+    /* Special ACL RHS "ipv6" matches IPv6 Internet */
     if (strcmp(t, "ipv6") == 0) {
         debugs(28, 9, "aclIpParseIpData: magic 'ipv6' found.");
         r = q; // save head of the list for result.
 
-        /* 0000::/4 is a mix of localhost and obsolete IPv4-mapping space. Not valid outside this host. */
+        /* RFC 4291 section 2.5.2. The Unspecified Address: ::/128 */
+
+        /* RFC 4291 section 2.5.3. The Loopback Address : ::1/128 */
```

Comment:
It was documenting that the range was considered, but not added to the set. Does the comment(s) added in [62a20b8](https://github.com/squid-cache/squid/pull/1303/commits/62a20b8507ab393755047590e4c5147e855c4ce2) solve this?

Reply:
I am resolving this change request because the current PR addressed its primary concern. Thank you. We may need to adjust the added comments, but that change (if any) can be tracked in a separate change request.